app/src/static_path.js

```diff
@@ -0,0 +1,40 @@
+
+/// Check if the code is running in Electron.
+function is_electron() {
+    // Renderer process
+    if (typeof window !== 'undefined' && typeof window.process === 'object' && window.process.type === 'renderer') {
+        return true;
+    }
+
+    // Main process
+    if (typeof process !== 'undefined' && typeof process.versions === 'object' && !!process.versions.electron) {
+        return true;
+    }
+
+    // Detect the user agent when the `nodeIntegration` option is set to true
+    if (typeof navigator === 'object' && typeof navigator.userAgent === 'string' && navigator.userAgent.indexOf('Electron') >= 0) {
+        return true;
+    }
```

Comment:
these lines are too long plus there is a lot of repetition. Moreover, this seems like a really complex way of chekcing if we are in electron. Is there an official doc telling that this is the way to check it? If so, it should be linked in a comment.